pkg/api/graph/test/overlapping-hpas.yaml

```diff
@@ -71,7 +71,7 @@ items:
     maxReplicas: 10 
     cpuUtilization:
       targetPercentage: 80
-- apiVersion: v1
+- apiVersion: apps.openshift.io/v1
```

Comment:
Did you have to change this?  Did only some of them change?  If you don't have to change these, consider changing all but one back.